cfme/tests/services/test_rest_services.py

```diff
@@ -820,8 +820,12 @@ def _order_finished():
 
         wait_for(_order_finished, num_sec=180, delay=10)
 
-        service_name = re.search(
-            r'\[({}[0-9-]*)\] '.format(template.name), service_request.message).group(1)
+        # Regex to parse "[EVM] Service [NAME I CARE ABOUT] Provisioned Successfully
+        # group 'name', non-greedy .*? to get the ending ]
+        # (?!{<appliance.server_name>}) this is to filter out [EVM] part from matches
+        regex_string = '\[(?P<name>(?!{})*?)\]'.format(appliance.server_name())
+
+        service_name = re.search(regex_string, service_request.message).group('name')
```

Comment:
Due to https://github.com/ManageIQ/integration_tests/pull/5516 the service names no longer contain timestamp and ``template.name``. The name of the service is now the same as the label of the dialog that corresponds to the service template.
The correct approach is to get the dialog label instead of using regex, IMHO.

Reply:
Thanks Martin - I was definitely still playing with things here, as a similar pattern is used in other places in the test. I'll check the available objects to get this name in a better way - possibly changing this to assert the expected name that we already have is in the `service_request.message`.